### PR TITLE
Add a detailed export to the action view of the event directory #3563

### LINKED ISSUE #3563: Add a detailled export in actions view with event description
### Situation Description

The actions view in the event directory only has a basic export.

### Feature Description

We wish to create a detailled export that includes the event description (this value will not be added as a column in the view).

@bernardsilenou @carolinverset

### PR DESCRIPTION
**Pull request by alexandre-gille**
Closes #3563

### CODE REVIEW

**Review by MateStrysewske — CHANGES_REQUESTED** [review]

File: sormas-api/src/main/java/de/symeda/sormas/api/event/EventActionExportDto.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+		this.eventInvestigationStatus = eventInvestigationStatus;
+		this.actionTitle = actionTitle;
+		this.actionCreationDate = actionCreationDate;
+		this.actionChangeDate = actionChangeDate;
+		this.actionStatus = actionStatus;
+		this.actionPriority = actionPriority;
+		this.actionReplyingUser = new UserReferenceDto(actionReplyingUserUuid, actionReplyingUserFirstName, actionReplyingUserLastName, null);
+	}
+
+	@Order(0)
```
[the remark this answers is not in the document]
Comment (alexandre-gille) [reply sormas-api/src/main/java/de/symeda/sormas/api/event/EventActionExportDto.java:78]: Oh, I didn't think devs were editing this by hand ! Whenever I have to modify the order I just use a sequence. But if this is really needed, i'll change this.

[the remark this answers is not in the document]
Comment (MateStrysewske) [reply sormas-api/src/main/java/de/symeda/sormas/api/event/EventActionExportDto.java:78]: I'm not familiar with that - how exactly are you doing it? Would definitely be better than having to do it by hand!

[the remark this answers is not in the document]
Comment (alexandre-gille) [reply sormas-api/src/main/java/de/symeda/sormas/api/event/EventActionExportDto.java:78]: Well I think there's this kind of feature on any major IDE (At least with a plugin). If you are using IntelliJ Idea you can use the String Manipulation plugin and use its 'create sequence' feature (there's also plenty of other useful feature for lazy people like me on this plugin).

[the remark this answers is not in the document]
Comment (MateStrysewske) [reply sormas-api/src/main/java/de/symeda/sormas/api/event/EventActionExportDto.java:78]: Fine by me to leave it like this then :)

**Review by MateStrysewske — APPROVED** [review]

**Pull request merged**